build_in_methods_iterators/homework.py

```diff
@@ -14,7 +16,15 @@ def task_1_fix_names_start_letter(data: DT) -> DT:
         fix_names_start_letters([{'name': 'Alex', 'age': 26}, {'name': 'denys', 'age': 89}])
         >>> [{'name': 'Alex', 'age': 26}, {'name': 'Denys', 'age': 89}]
     """
-    pass
+    for member in data:
+        if 'name' in member:
+            member["name"] = member["name"].capitalize()
+    return data
```

Comment:
use in list comprehension dictionary method update

Reply:
done!